packages/create-vite/template-ember-ts/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": "@tsconfig/ember",
+  "glint": {
+    "environment": ["ember-template-imports"]
+  },
+  "compilerOptions": {
+    "allowImportingTsExtensions": true,
+    // Stylistic
+    "noPropertyAccessFromIndexSignature": false,
+    // Ecosystem
+    "declaration": false,
+    "declarationMap": false,
```

Comment:
Are these fields enabled by @tsconfig/ember? They should not be needed

Reply:
They are presently needed. I've opened a PR:  https://github.com/tsconfig/bases/pull/303

Reply:
removed customizations here -- we published our own tsconfig